gateleen-hook/src/main/java/org/swisspush/gateleen/hook/HttpHook.java

```diff
@@ -98,9 +98,9 @@ public void setExpireAfter(int expireAfter) {
     }
 
     /**
-     * Returns the expiration time of this hook.
-     * 
-     * @return expirationTime
+     * @return
```

Comment:
Since this method now can return _null_ values, I would make it more clear to the user, that the value could be _null_. This could be achieved by returning a _java.util.Optinal<LocalDateTime>_ object.

Example:
```java
public Optional<LocalDateTime> getExpirationTime() {
	return Optional.ofNullable(expirationTime);
}
```

Reply:
or you write it to the JavaDoc...

Reply:
This seems to be _gateleen public API_. Therefore keep in mind that we introduce a _breaking change_ doing this.

Nevertheless I'll apply this  :-)